src/brain_teasers/python/deletions_to_make_valid_parentheses/tests/test_deletions_to_make_valid_parentheses.py

```diff
@@ -1,8 +1,12 @@
 import unittest
 
-from src.deletions_to_make_valid_parentheses import (
```

Comment:
did you try to include  this directory in the PYTHONPATH environment ? there are some solutions here : [stackoverflow](https://stackoverflow.com/questions/51049663/python3-6-error-modulenotfounderror-no-module-named-src)